requirements-tests.txt

```diff
@@ -9,3 +9,5 @@ flake8-pyi==22.5.1
 # must match .pre-commit-config.yaml
 isort==5.10.1
 tomli==1.2.2
+packaging==21.3
+termcolor
```

Comment:
I'm fine with this, but can you delete this section in CONTRIBUTING, since it's no longer an "optional dependency" if we do this: https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#optional-dependencies